course_content/notebooks/cartopy_intro.ipynb

```diff
@@ -148,8 +148,44 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Adding features\n",
+    "## Adding features\n",
     "\n",
+    "We can add features from the [Natural Earth](http://www.naturalearthdata.com/) database to maps we produce. Natural Earth datasets are downloaded and cached when they are first used, so you need an internet connection to use them, and you may encounter warnings when you first download them.\n",
+    "\n",
+    "We add features to maps via the [cartopy feature interface](http://scitools.org.uk/cartopy/docs/latest/matplotlib/feature_interface.html).\n",
+    "\n",
+    "For example, let's add political borders to a map:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import cartopy.feature as cfeat\n",
+    "\n",
+    "fig = plt.figure(figsize=(14, 7))\n",
+    "\n",
+    "ax = plt.axes(projection=ccrs.Miller())\n",
+    "ax.coastlines('50m')\n",
+    "\n",
+    "# ax.add_feature(cfeat.BORDERS, edgecolor='r')\n",
+    "\n",
+    "political_bdrys = cfeat.NaturalEarthFeature(category='cultural',\n",
+    "                                            name='admin_0_countries',\n",
+    "                                            scale='50m')\n",
+    "ax.add_feature(political_bdrys, edgecolor='r', facecolor='none', zorder=-1)\n",
```

Comment:
Colour and lat long limit suggestion to improve visualisation of borders:
`ax.add_feature(political_bdrys, edgecolor='black', facecolor=cfeat.COLORS['land'], zorder=-1)
ax.set_xlim(-15,45)
ax.set_ylim(35,70)`